src/Scenes/GameScene/LevelRenderer.ts

```diff
@@ -4,6 +4,8 @@ import {TextureFactory} from '../../../src.common/Managers/TextureFactory';
 import {Level} from '../../GameLogic/Level';
 import {FloorType} from '../../GameLogic/Enums';
 import {GfxConstants} from '../../Core/Constants/GfxConstants';
+import Constants from '../../Core/Constants';
```

Comment:
Should there be { } around Constants here?

Reply:
No, because the that file does a default export. You could put anything in place of `Constants` and that would know it refers to the default export in that file (spoken from memory).